src/components/date-picker/date-picker.tsx

```diff
@@ -462,9 +504,10 @@ export class DatePicker implements LocalizedComponent {
         <calcite-date-picker-month-header
           activeDate={activeDate}
           headingLevel={this.headingLevel || HEADING_LEVEL}
-          intlNextMonth={this.intlNextMonth}
-          intlPrevMonth={this.intlPrevMonth}
-          intlYear={this.intlYear}
+          intlNextMonth={this.messages.nextMonth}
+          intlPrevMonth={this.messages.prevMonth}
+          intlYear={this.messages.year}
+          lang={this.effectiveLocale}
```

Comment:
We don't need to set lang here nor pass the locale, right?

Reply:
Nope, we don't. It slipped through while merging locally. Good catch.